### DISCUSSION TITLE
New version is maybe better as a general EU layout, worse for everyday FR use

### DISCUSSION THREAD

**Discussion by oKcerG** [qwerty-fr/qwerty-fr#87] [General]
Hello, first I'd like to thank you for this great project making qwerty keyboards usable and arguably better than azerty keyboards.

However the recent changes has made this layout worse for me than it was before.
As a french coder I liked this layout because it made access to frequently used accentuated letters easy and intuitive while still keeping the advantage of qwerty keyboards (easily accessible numbers, parentheses, brackets, braces, semi-colon, dot, etc) and not deviating too much from qwerty us int.

Now it is not quite easy as before :
- No easy access to °, to £
- No access to ³
- I preferred where the € was since it matched the other QWERTY layouts (and possibly what is shown on the physical keycap)
   
The new changes made it easier for things I will never use, like å q̧ ȳ (but I don't doubt some will).

Would it be sensible to fork this project in 2?

I guess I can fork it myself and revert some 0.7 changes, but the new project aim doesn't seem to match with the name qwerty-fr anymore.
Also the following sentence in the readme is incorrect if we consider the US Int alt-gr keys like ³, €, £
> This layout is a strict superset of the QWERTY layout. This means that all the keys of the QWERTY layout are kept in the same location. Anyone using a QWERTY layout can type on the QWERTY-fr layout without even knowing that they are not typing on a QWERTY layout.

**Comment by devnoname120** [comment]
I will update the `README.md` later but the keyboard is neither based nor meant to be based on `US Intl` or any UK flavors. It's instead based on [`US ANSI/ISO (terminal style)`](https://www.farah.cl/Keyboardery/A-Visual-Comparison-of-Different-National-Layouts/#enUS):

<img width="618" alt="image" src="https://github.com/qwerty-fr/qwerty-fr/assets/2824100/bd546798-7822-4371-a654-0bf3bd56c9bc">

<img width="617" alt="image" src="https://github.com/qwerty-fr/qwerty-fr/assets/2824100/c01fac6f-34a3-45a5-b34c-afbed53b72c0">

I'm open to changing the base to `US ANSI/ISO (PC style)` if someone feels strongly in favor of it. It only changes the key next to the left shit key and it hasn't been requested so far:
<img width="620" alt="image" src="https://github.com/qwerty-fr/qwerty-fr/assets/2824100/455fbdf0-14ee-4b52-a1eb-cdb4afea51d3">

I will address your other points later.

**Comment by lcoulet** [comment]
Thanks a lot. Unfortunately the most common in Europe are UK ISO keyboards, all laptops I have are UK ISO. Maybe can this be made configurable ? If you explain how to change keymaps (I am stupid) I would be happy to make a fork :-)

**Comment by Laurent-PRP** [comment]
Hello @devnoname120,
Regarding your last answer, I think that supporting `US ANSI/ISO (PC style)` instead of `US ANSI/ISO (terminal style)` would be great, because it will still be 100% compatible with the other.

Today if I want to type ≥ or ≤ with the PC style keyboard, I can't. If they were moved to another key they would be available on both keyboard.

But it's a minor concern, the tool is still great either way.

**Comment by L-Naej** [comment]
Just to add to the initial post: I had the same thought regarding the **î** and **ì**. ì is never used in French so it's very strange for a qwerty-fr layout to bind it to _alt gr + i_ and to bind î on _alt gr + 8_. 
The "logical" binding would be the opposite: î bound to alt gr + i.
Edit: same thing for ò which is not a french character but bound on altgr + o. ô would be the logical binding.

Thanks for this layout though, very happy that it exists!
